packages/babel-plugin-transform-runtime/src/runtime-corejs3-definitions.js

```diff
@@ -168,16 +230,48 @@ export default () => {
         iterator: { stable: true, path: "symbol/iterator" },
         keyFor: { stable: true, path: "symbol/key-for" },
         match: { stable: true, path: "symbol/match" },
+        matchAll: { stable: false, path: "symbol/match-all" },
```

Comment:
`symbol/match-all` and `symbol/replace-all` available only from `core-js@3.1`, so it's also a breaking change for a case of old dependencies.